packages/convex-helpers/README.md

```diff
@@ -623,10 +623,266 @@ export const lastCountLongerThanName = query({
   handler: async (ctx) => {
     return await filter(
       ctx.db.query("counter_table"),
-      (c) => c.counter > c.name.length,
+      (c) => c.counter > c.name.length
     )
       .order("desc")
       .first();
   },
 });
 ```
+
+## Manual Pagination
+
+### Benefits of manual pagination
+
+- You can preprocess the table to store all page boundaries, allowing skipping to an
+  arbitrary page.
```

Comment:
yeah i was thinking it could be a component. i was thinking you could do it with function references, where the passed-in function takes in a start index key, calls `getPage`, and returns the end index key. but the write-time update hook is more complex